app/views/guidance/claim-guidance-for-farmers.njk

```diff
@@ -66,7 +66,7 @@
             <li>date when the vet last visited the farm to carry out the review</li>
             <li>date the vet completed endemic disease or condition testing</li>
             <li>vet’s name</li>
-            <li>vet’s RCVS number</li>
+            <li>Vet's Royal College of Veterinary Surgeons (RCVS) Number</li>
```

Comment:
Although the ticket spells out the exact text, I think that the initial "Vet's" should be all lowercase to match with the other bullet points

Reply:
![Uploading image.png…]()